redirect-rules/simple/README.md

```diff
@@ -0,0 +1,74 @@
+# Redirect Rule: Simple Example with a Webtask
+
+This sample demonstrates a minimal implementation of the `redirect` protocol. It contains an Auth0 [rule](http://auth0.com/docs/rules) that will redirect the user to a simple consent form. If the user checks the "I agree" checkbox on this form and clicks the **Submit** button, they are then redirected back to Auth0 to complete the authentication flow. On future logins they will no longer be prompted since the consent action is stored in their user profile.
+
+The consent form is hosted using a simple [Webtask](https://webtask.io/) that you can easily modify and provision and use in your version of the rule.
+
+## Rule Setup
+
+To try this rule out with your own Auth0 account using an existing instance of the consent form webtask, follow these steps:
+
+1. Create a new Auth0 rule using the contents of the [`rule.js`](rule.js) script.
+1. In your account's [Rules](https://manage.auth0.com/#/rules) screen, add a setting that has the key `CONSENT_FORM_URL` and the following value:  
+  ```
+  https://sandbox.it.auth0.com/api/run/simple-redirect-protocol/consent?webtask_no_cache=1
```

Comment:
@tjanczuk: The README perhaps isn't very clear on this point, but further down I do demonstrate creating a custom redirect URL to a copy of the webtask that we show here. These steps are just a means to quickly try out a very simple redirect rule without having to create and host your own 3rd party website. But I will make this more clear in the README.

I do like your idea of passing back more than just the value of a checkbox. To align more the example, perhaps we can also send back the end user's initials as a means of signing the consent form.

I'm not sure what you mean by:
> this rule is pre-provisioned by us
The rule code has to by copy/pasted from the `rules` repo to a new rule in a customer's tenant.